hosting/static/hosting/js/virtual_machine_detail.js

```diff
@@ -90,26 +89,28 @@ $(document).ready(function() {
             success: function (data) {
                 if (data.status === true) {
                     fa_icon = $('.modal-icon > .fa');
-                    fa_icon.attr('class', 'fa fa-check');
-                    $('.modal-header > .close').attr('class', 'close');
+                    fa_icon.attr('class', 'checkmark');
+                    // $('.modal-header > .close').removeClass('hidden');
                     $('#createvm-modal-title').text(data.msg_title);
                     $('#createvm-modal-body').text(data.msg_body);
-                    $('#createvm-modal').on('hidden.bs.modal', function () {
-                        window.location = data.redirect;
```

Comment:
In the PR #493 which is yet to be merged, we seem to be using the same modal both for landing (user not logged in) and hosting (user logged in) cases. First the first case: on successful vm creation, `data.redirect` contains url to datacenterlight's index page, while for the logged in users, we take the user to his/her virtual machines page. I think the first case may not be covered in this PR. So, to be generic for both logged in and not logged in cases, I propose that we reuse `data.redirect`.

Reply:
ok, didn't know that, reverting this back 👍